.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
If we don't copy/mount the `node_modules` in the container we have to run `npm` on the host machine? Or what does this file do? 🤔

Reply:
No, When we build images at that time only it will be ignored, so we have small image, I believe does that only.. We can still run commands inside container and it should be available, but let me confirm it one more time

Reply:
@Gummibeer - correct me if I'm wrong but you wanted this only for local development right? If so then I'm not sure the images will be used for anything other than local artifacts.

It seems the docker-compose already mounts the local project folder as `/var/www` for both containers...so maybe just skip the `.dockerignore` all together since the local files will be mounted to both and any images won't be published?

Could probably even drop line 5 of the dockerfile for the nginx container too. IMO there's no need to copy files into an image layer's `/var/www`, right before we mount the local system directory into the containers `/var/www`.

Reply:
@mallardduck You are correct about copying files in nginx container not needed, Actually I was doing some specific changes (adding some resolvers and upstreams) when in local dev env so added that but later i removed it to keep things simple for now, so forgot to remove that line 5.. but it is removed now.. 😉

now question is i am not sure what do you mean by skipping `.dockerignore` ? why do you want to skip it ?  🧐

Reply:
From my understanding the `.dockerignore` file is only used for `ADD` or `COPY` commands in a `Dockerfile`.
Because we're just using this for local dev and not image building these actions aren't being used in either `Dockerfile`.

Reply:
It's right that the docker is only for local development.
None of the images will leave the developers local machine. Production still runs with native installed things on normal VMs

Reply:
what do you suggest @Gummibeer  ?

Reply:
I have no idea what the docker ignore does and in which moment of build process or whatever it's used. But as we only use them locally and mount everything anyway I don't see what that file could provide/improve except for confusion!? 🤔

Reply:
My experience with `.dockerignore` is that I use it to prevent files getting into the published images.
So when I want to package my app itself into an image, push that image to a remote, then pull into a prod.
That way to 'deploy' the app to k8s -for instance- I'd grab my apps image, add the env vars - then let the container start script do the rest.

My vote is to just remove the file from this PR. 😄

Reply:
@mallardduck from your comment i see no harm in adding, and no benefit in removing.. 🧐

in this situation my vote is for to keep it..

Reply:
I have no objective opinion but in general I tend more to agree with Dan in that case.
If the file isn't needed/doesn't help with that setup and its usage purpose I would like to don't have it in at all. As every file only increases mental complexity and I and possibly others will always ask "What is that file for?"

Reply:
That file helps reduce build images sizes.. a person like me who has lot of projects and docker images to work with needs smaller sizes.. but anyways will remove it in next push .. 👍🏻